src/code/providers/google-drive-provider.coffee

```diff
@@ -12,11 +12,27 @@ GoogleDriveAuthorizationDialog = React.createFactory React.createClass
   displayName: 'GoogleDriveAuthorizationDialog'
 
   getInitialState: ->
+    @_hasLoadedGAPI = false
```

Comment:
I was following the pattern I used in `AuthorizeMixin`, but what you suggest would be simpler so I'll check it out.